modules/oe_webtools_maps/src/Plugin/Field/FieldFormatter/WebtoolsMapFormatter.php

```diff
@@ -64,17 +73,51 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
     $element = [];
 
     foreach ($items as $delta => $item) {
+      $data_array = [
+        'service' => 'map',
+        'version' => '2.0',
+        'map' => [
+          'zoom' => $this->getSetting('zoom_level'),
+          'center' => [$item->get('lat')->getValue(), $item->get('lon')->getValue()],
+        ],
+      ];
+
+      if ($this->getSetting('show_marker')) {
+        $data_array['layers'] = [
+          [
+            'markers' => [
+              'type' => 'FeatureCollection',
+              'features' => [
```

Comment:
This is not related to this PR, but I find it strange that the `markers` property seems to only accept 1 single marker? Is this how the API expects the data? How would we be able to show more than one marker using this data structure?